File: PYTHON_PRACTICE/leap_year.py
#CHECKING WHETHER A YEAR IS A LEAP YEAR OR NOT
year=int(input("enter a year:"))
if year%4==0:
    print("leap year")
elif year %400==0:
    print("leap year")
elif year%100==0:
    print(" not a leap year")
else:
    print("not a leap year");

File: JAVA_PRACTICE/assignment1/Leap_year.java
import java.util.*;
class Leap_year
{
 public static void main(String args[])
  {
   Scanner sc=new Scanner(System.in);
   int a;
   System.out.println("enter the year:");
   a=sc.nextInt();
   if((a%4==0) && (a%100!=0))||(a%400==0))
   {
    System.out.println("leap year");
   }
   else
   {
   System.out.println("Not a leap year");
   }
  }
 }
   